src/data/faq_de.json

```diff
@@ -549,6 +549,14 @@
                                     "Die Spezifikationen der EU für Impfzertifikate für Auffrischimpfungen wurden geändert. Sollte Ihr Zertifikat nicht mehr den aktuellen Spezifikationen entsprechen, ist es zwar weiterhin gültig, es kann jedoch sein, dass bei einer Prüfung die Auffrischimpfung nicht als solche erkannt wird. In solch einem Fall können Sie über die Corona-Warn-App ab Version 2.20.3 mit einer entsprechenden Nachricht darüber informiert werden. Sie können dann freiwillig ein neues Impfzertifikat kostenlos über die App anfordern. Nachdem Sie Ihr Einverständnis gegeben haben, übermittelt die App das Impfzertifikat zur Überprüfung an das RKI, welches ein aktualisiertes Zertifikat ausstellt und an Ihre App übermittelt. Sie finden das aktualisierte Zertifikat dann in der Zertifikate-Registerkarte. Das alte Zertifikat wird automatisch in den Papierkorb verschoben und nach 30 Tagen gelöscht. Das alte Zertifikat kann innerhalb der 30 Tage jederzeit wiederhergestellt werden.",
                                     "Weitere Informationen finden Sie im Blog: <a href='../../blog/2022-03-30-cwa-2-20/' target='_blank'>Corona-Warn-App Version 2.20</a>"
                                 ]
+                            },
+                            {
+                                "title": "Ich erhalte die Meldung, dass die Gültigkeit eines Zertifikats bald abläuft. Was muss ich tun?",
+                                "anchor": "dcc_replacement_howto",
+                                "active": false,
+                                "textblock": [
+                                    "Die technische Gültigkeit der digitalen COVID-Zertifikate läuft nach 365 Tagen automatisch ab. Für alle relevanten Zertifikate erhalten Sie dazu 28 Tage vor Ablauf eine Mitteilung in der Corona-Warn-App. Um Ihren Impfstatus weiterhin mit der App nachweisen zu können, müssen die entsprechenden Zertifikate aktualisiert werden. Die dazu notwendige Neuausstellung können Sie demnächst in der App mit wenigen Klicks selbst durchführen."

```

Comment:
"Für alle relevanten Zertifikate erhalten Sie dazu 28 Tage vor Ablauf eine Mitteilung in der Corona-Warn-App."

Please make sure this is true, see https://github.com/corona-warn-app/cwa-wishlist/issues/828#issuecomment-1132110493.

Reply:
@dsarkar Why was this review comment not addressed?

Reply:
@Ein-Tim OK, let me cross-check.

Reply:
Thank you!

Reply:
Just a minor inconsistency, if I understood @thomaugsten correctly, currently, there are no notifications reg. expired certificates at all.